include/dxc/HLSL/DxilPipelineStateValidation.h

```diff
@@ -71,6 +71,10 @@ struct PSVRuntimeInfo1 : public PSVRuntimeInfo0
 {
   uint8_t ShaderStage;              // PSVShaderKind
   uint8_t UsesViewID;
+  union {
+    uint16_t MaxVertexCount;          // MaxVertexCount for GS only (max 1024)
+    uint8_t SigPatchConstantVectors;  // Output for HS; Input for DS
```

Comment:
Tests will be in another PR.  No more changes can be made for this PR, as this goes in dxil-2017-06 as-is.  I will add the recommended change along with the test PR when ready.